cmd/juju/application/deploy.go

```diff
@@ -623,7 +633,7 @@ var (
 // whether we are deploying a charm or a bundle.
 func charmOnlyFlags() []string {
 	charmOnlyFlags := []string{
-		"bind", "config", "constraints", "force", "n", "num-units",
+		"bind", "config", "constraints", "n", "num-units",
```

Comment:
don't we need "force" back in this slice?

Reply:
Yes!